kairon/api/app/routers/bot/data.py

```
@@ -491,3 +519,23 @@ async def get_pos_endpoint(
     """
     integration_endpoint = cognition_processor.get_pos_integration_endpoint(current_user.get_bot(), provider, sync_type)
     return Response(data=integration_endpoint, message="Endpoint fetched", success=True, error_code=0)
+
+@router.post("/collection/bulk/{collection_name}", response_model=Response)
+async def save_bulk_collection_data(
+    request: BulkCollectionDataRequest,
+    collection_name: str = Path(..., description="Collection name"),
+    current_user: User = Security(Authentication.get_current_user_and_bot, scopes=DESIGNER_ACCESS),
```

Ruff (0.12.2)

Do not perform function call `Security` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)